components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/header.adoc

```diff
@@ -3,32 +3,44 @@
 [id="block_structure"]
 = Block structure
 
-In Starknet, a block is a list of transactions, and a block header that contains the following fields:
+A Starknet block is a list of transactions and a block header that contains the following fields:
 
 
 [%autowidth]
 |===
-| Name | Type | Description | Implemented
-
-| `parent_block_hash` | `FieldElement` | The hash of this block's parent | &#10003;
-|`block_number` | `Integer` | The number (height) of this block | &#10003;
-| `global_state_root` | `FieldElement` | The state xref:../Network_Architecture/starknet-state.adoc#state_commitment[commitment] after this block | &#10003;
-|`sequencer_address` | `FieldElement` | The Starknet address of the sequencer who created this block | &#10003;
-| `block_timestamp` | `Timestamp` | The time the sequencer created this block before executing transactions | &#10003;
-|`transaction_count` | `Integer` | The number of transactions in a block | &#10003;
-| `transaction_commitment` | `FieldElement` | A commitment to the transactions included in the block | &#10003;
-|`event_count` | `Integer` | The number of events | &#10003;
-| `event_commitment` | `FieldElement` | A commitment to the events produced in this block | &#10003;
-| `protocol_version` | `Integer` | The version of the Starknet protocol used when creating this block |
-| `extra data` | `FieldElement` | Extraneous data that might be useful for running transactions |
-|===
+| Name | Type | Description
+
+| `parent_block_hash` | `FieldElement` | The hash of the block's parent.
+|`block_number` | `Integer` | The number, that is, the height, of this block.
+| `global_state_root` | `FieldElement` | The xref:../Network_Architecture/starknet-state.adoc#state_commitment[state commitment] after the block.
+|`sequencer_address` | `ContractAddress` | The Starknet address of the sequencer that created the block.
+| `block_timestamp` | `Timestamp` | The time the sequencer created the block, before executing transactions, in seconds since the Unix epoch.
+|`transaction_count` | `Integer` | The number of transactions in the block.
+| `transaction_commitment` | `FieldElement` | A commitment to the transactions included in the block.
+
+The root of a 64-bit high binary Merkle Patricia trie. The leaf at index stem:[$i$] corresponds to stem:[$${h(transaction \space hash, signature)}$$] if the stem:[$i'th$] transaction is an `invoke` transaction and stem:[$h(0,0)$] otherwise.
```

Comment:
```suggestion
The root of a 64-bit high binary Merkle Patricia trie. The leaf at index stem:[$i$] corresponds to stem:[$${h(\text{transaction_hash}, \text{signature})}$$] if the stem:[$i'th$] transaction is an `invoke` transaction and stem:[$h(0,0)$] otherwise.
```